star_tides/services/databases/mongo/models/__init__.py

```diff
@@ -0,0 +1,3 @@
+''' star_tides.services.mongo.models
+'''
+import mongoengine as me
```

Comment:
What does this do?

Reply:
Nothing really. In sqlalchemy you need to define a reference to the database object because you have to mess with the session_scope and such so a declaration like this is necessary. That doesn't seem to be the case with mongoengine.